cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/metric/MetricsUtils.java

```diff
@@ -228,6 +237,8 @@ private static CruiseControlMetric toCruiseControlMetric(long now,
       case SIZE:
         int partition = Integer.parseInt(tags.get(PARTITION_KEY));
         return new PartitionMetric(MetricType.PARTITION_SIZE, now, brokerId, topic, partition, value);
+      case REQUEST_HANDLER_AVG_IDLE_PERCENT:
+        return new BrokerMetric(MetricType.BROKER_REQUEST_HANDLER_AVG_IDLE_RATE, now, brokerId, value);
```

Comment:
It is a "ratio" instead of a "rate". Also it is probably better to keep the name the same as the original metric name, i.e. "BROKER_REQUEST_HANDLER_AVG_IDLE_PERCENT"